src/library/basetracktablemodel.h

```diff
@@ -16,6 +16,9 @@ class BaseTrackTableModel : public QAbstractTableModel, public TrackModel {
     DISALLOW_COPY_AND_ASSIGN(BaseTrackTableModel);
 
   public:
+    // This role is used for data export like in CSV files
+    static constexpr int kDataExportRole = Qt::UserRole + 1;
```

Comment:
The base class TrackModel already defines constants for. Qt::UserRole + 0 and Qt::UserRole + 1. Potential conflict?

Reply:
Not really, because it is used is a different context. However, I have moved it to TrackModel anyway, to avoid future confusion.